### PR TITLE
dd to delete files

### PR DESCRIPTION
**Pull request by itzjustalan**
# dd to delete files
# move/reorder filenames
# more granular control with lines numbers
# include dot files

### CODE REVIEW

**Comment by motiejus** [comment]
FYI refs #17

File: vimv [hunk 1]
```diff
@@ -12,28 +16,38 @@ trap '{ rm -f "${FILENAMES_FILE}" ; }' EXIT
 if [ $# -ne 0 ]; then
     src=( "$@" )
 else
-    IFS=$'\r\n' GLOBIGNORE='*' command eval 'src=($(ls))'
```
Comment (motiejus) [inline vimv:15]: perhaps this should be in a separate pull request

Comment (itzjustalan) [reply vimv:15]: if somebody wants this feel free.
ive already made it to what i want
